examples/use-cases/log-msgs/go/main.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/nats-io/nats.go"
+)
+
+func main() {
+	natsURL := os.Getenv("NATS_URL")
+	if natsURL == "" {
+		natsURL = nats.DefaultURL
+	}
+
+	nc, _ := nats.Connect(natsURL)
+
+	js, _ := nc.JetStream()
+
+	js.AddStream(&nats.StreamConfig{
+		Name:      "EVENTS",
+		Subjects:  []string{"events"},
+		Storage:   nats.MemoryStorage,
+		Retention: nats.WorkQueuePolicy,
+	})
+
+	js.AddConsumer("EVENTS", &nats.ConsumerConfig{
+		Durable:   "PROCESSOR",
+		AckPolicy: nats.AckExplicitPolicy,
+	})
+
+	/* Push...
+	js.AddConsumer("EVENTS", &nats.ConsumerConfig{
+		Durable:        "PROCESSOR",
+		AckPolicy:      nats.AckExplicitPolicy,
+		DeliverSubject: "processor.events",
+		DeliverGroup:   "processor",
+	})
+	*/
+
+	ctx := context.Background()
+
+	// Context to stop publishers after an explicit amount of time.
+	pctx, pcancel := context.WithTimeout(ctx, 2*time.Second)
+	defer pcancel()
+	startPublishers(pctx, nc, 250000, "events", 200)
```

Comment:
Emulate 250k msgs/sec as best we can for two seconds (just for the sake of the example).. with 200 byte payloads.